data/domain/world-6/farming.tsx

```diff
@@ -408,10 +520,11 @@ export class Farming extends Domain {
         const farming = data.get(this.dataKey) as Farming;
         const cropsData = data.get("FarmCrop") as Object;
         const plotsData = data.get("FarmPlot") as number[][];
+        const landRankData = data.get("FarmRank") as number[][];
         const upgradesData = data.get("FarmUpg") as number[];
         
         // Old accounts won't have this data, exit early.
-        if (!cropsData || !plotsData || !upgradesData) {
+        if (!cropsData || !plotsData || !upgradesData|| !landRankData) {
```

Comment:
This check doesn't seem to actually work, potentially because of my data parsing changes these now come back as an empty array instead of undefined.

So need to fix this to look at length of 0 maybe?

Reply:
I updated this to also check length